src/platform/forms-system/src/js/utilities/data/profile.js

```diff
@@ -284,6 +292,13 @@ export const getMissingInfo = ({ data, keys, content, requiredKeys = [] }) => {
   return missingInfo.filter(Boolean);
 };
 
+/**
+ *
+ * @param {function[]} validations - validations array from ui:validations in uiSchema
+ * @returns {string[]}
+ */
+export const getValidationErrors = validations => checkValidations(validations);
```

Comment:
Is there a need to declare `getValidationErrors` separately from `checkValidations`?

Reply:
You're right! I had this function, but forgot to delete this code after removing it from the `ContactInfo` component